web/src/scheduled_messages_ui.js

```diff
@@ -62,10 +62,7 @@ export function open_scheduled_message_in_compose(scheduled_msg, should_narrow_t
         narrow_via_edit_scheduled_message(compose_args);
     }
 
-    compose.clear_compose_box();
```

Comment:
We clear the compose in `start` after saving the draft if needed, hence this line was not just redundant but buggy.